e2e/features/data/scanned-case.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  formData : [
+    { question: 'is_hearing_type_oral', answer: 'true' },
+    { question: 'is_hearing_type_paper', answer: 'false' },
+    { question: 'person1_first_name', answer: 'cde' },
+    { question: 'person1_last_name', answer: 'def' },
+    { question: 'person1_address_line1', answer: 'efg' },
+    { question: 'person1_address_line3', answer: 'fgh' },
+    { question: 'person1_address_line4', answer: 'ghi' },
+    { question: 'person1_postcode', answer: 'hij' },
+    { question: 'person1_nino', answer: 'ijk' }
```

Comment:
I think mrn_date has now been added as a mandatory field

Reply:
brilliant, I'll add that, thanks